NeuroID/src/advancedDeviceLib/java/com/neuroid/tracker/extensions/AdvancedDeviceExtension.kt

```diff
@@ -19,20 +19,13 @@ import com.neuroid.tracker.utils.Constants
  * Return false if not started.
  */
 fun NeuroIDPublic.start(advancedDeviceSignals: Boolean): Boolean {
+    NeuroID.getInternalInstance()?.isAdvancedDevice = advancedDeviceSignals
```

Comment:
This function should NOT change the global variable, it should only override the global variable

Reply:
not changing the global variable for now. will convene to simplify the flow later.